broker/l2tp_broker.py

```diff
@@ -792,7 +792,15 @@ def hook(self, name, *args):
     # Execute the registered hook
     logger.debug("Executing hook '%s' via script '%s %s'." % (name, script, str([str(x) for x in args])))
     try:
-      gevent_subprocess.call([script] + [str(x) for x in args])
+      script_process = gevent.subprocess.Popen([script] + [str(x) for x in args], stdout=gevent.subprocess.PIPE, stderr=gevent.subprocess.PIPE)
```

Comment:
This will not work with legacy `gevent_subprocess`. Can you replace `gevent.subprocess` with `gevent_subprocess` (we detect the proper module at the top)?